test/services/keyVault/kv-test-utils.js

```diff
@@ -0,0 +1,267 @@
+  var propNames = Object.keys(key);
+  for (var i = 0; i < propNames.length; ++i) {
+    key[propNames[i]].__proto__ = BigInteger.prototype;
+  }
+  return key;
+}
+
+exports.setRsaParameters = function(dest, key) {
+  dest.n = bigIntegerToBuffer(key.n);
+  dest.e = bigIntegerToBuffer(key.e);
+  dest.d = bigIntegerToBuffer(key.d);
+  dest.p = bigIntegerToBuffer(key.p);
+  dest.q = bigIntegerToBuffer(key.q);
+  dest.dp = bigIntegerToBuffer(key.dP);
+  dest.dq = bigIntegerToBuffer(key.dQ);
+  dest.qi = bigIntegerToBuffer(key.qInv);
+};
+
+function bigIntegerToBuffer(n) {
+
+  // Convert to binary and remove leading zeroes.
+  var data = n.toByteArray();
+  var leadingZeroes = 0;
+  while (leadingZeroes < data.length && data[leadingZeroes] === 0) {
+    ++leadingZeroes;
+  }
+  if (leadingZeroes) {
+    data = data.slice(leadingZeroes);
+  }
+  return new Buffer(data);
+}
+
+function xdescribe(a,b,c) {
```

Comment:
Nobody else uses skip. I tried it and it polutes output with "pending", so can I leave the xdescribe?